pyproject.toml

```diff
@@ -6,8 +6,13 @@ line-length = 100
 target-version = ['py36', 'py37', 'py38', 'py39']
 
 [tool.cibuildwheel]
-manylinux-x86_64-image = "manylinux2010"
-manylinux-i686-image = "manylinux2010"
+manylinux-x86_64-image = "manylinux2014"
+manylinux-i686-image = "manylinux2014"
```

Comment:
I think this is implicit in cibuildwheel > 2.3 which is why I avoided upgrading it earlier. While we're updating things I think it's good to bump our cibuildwheel version too. It's fine to keep this here too though and be explicit about the version we're using, but maybe we should pull in the latest cibuildwheel release now too.

Reply:
I've pushed up a new commit to bump the version of cibuildwheel, but I've left this explicit setting of the `manylinux` images in place. It's probably clearer for the team in general if we're explicit, rather than relying on the default behaviour.